pkg/logs/input/docker/launcher.go

```diff
@@ -212,19 +281,49 @@ func newOverridenSource(standardService, shortName string, status *config.LogSta
 
 // startTailer starts a new tailer for the container matching with the source.
 func (l *Launcher) startTailer(container *Container, source *config.LogSource) {
+	if l.tailFromFile {
+		l.scheduleFileSource(container, source)
+	} else {
+		l.startSocketTailer(container, source)
+	}
+}
+
+func (l *Launcher) scheduleFileSource(container *Container, source *config.LogSource) {
 	containerID := container.service.Identifier
-	if _, isTailed := l.getTailer(containerID); isTailed {
+	if _, isTailed := l.fileSourcesByContainer[containerID]; isTailed {
 		log.Warnf("Can't tail twice the same container: %v", ShortContainerID(containerID))
 		return
 	}
-
 	// overridenSource == source if the containerCollectAll option is not activated or the container has AD labels
-	overridenSource := l.overrideSource(container, source)
+	fileSource := l.getFileSource(container, source)
```

Comment:
I think this comment needs to be updated, `overridenSource` does not exist in this context